api/src/pcapi/routes/pro/offers.py

```diff
@@ -245,9 +258,10 @@ def patch_offer(
                 withdrawalDelay=update_body.get("withdrawalDelay", offers_api.UNCHANGED),
                 withdrawalDetails=update_body.get("withdrawalDetails", offers_api.UNCHANGED),
                 withdrawalType=update_body.get("withdrawalType", offers_api.UNCHANGED),
+                shouldSendMail=update_body.get("shouldSendMail") or False,
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏
```suggestion
                shouldSendMail=update_body.get("shouldSendMail", False)
```

Reply:
Je me demande si dans ce cas on ne préfère pas justement laisser None, quitte à ce que ça casse quelque chose et que ça oblige à corriger
ça peut masquer un problème

Mais bon low logaf, si c'est `None` on retourne `False` me parait cohérent aussi

Reply:
Ok pour moi :+1: